### PR TITLE
Documentation on design schedule feature in dg

### PR DESCRIPTION
**Pull request by emilysim00**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -209,7 +209,58 @@ _to be updated_
 ---
 
 ### Schedule-related features
-_to be updated_
+![ScheduleUML](uml/classDiagrams/images/scheduleComponent.png)
+
+User are able to create and make changes to a 7-day workout plan schedule using the WerkIt application. For each day, user are only allowed
```
Comment (alanlowzies) [inline docs/DeveloperGuide.md:214]: Perhaps the word 'User' can be changed to its plural form 'Users'?

File: docs/uml/classDiagrams/ScheduleClass.puml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+class ScheduleCommand {
++ execute():void
+}
+
+
+
+class UI {
++ getUserInput()
++ printScheduleFileNotFoundMessage():void
++ printScheduleFileCreatedMessage():void
```
Comment (alanlowzies) [inline docs/uml/classDiagrams/ScheduleClass.puml:45]: I notice that you have added these two `UI` methods that are used by `FileManager`. Perhaps you can add a dependency line from `FileManager` to `UI`?

**Review by alanlowzies — COMMENTED** [review]
Looks alright. Some suggestions for your consideration. Thank you

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (emilysim00) [reply docs/DeveloperGuide.md:214]: Thanks for pointing that out!

**Pull request merged**